### PR TITLE
(#254) update the README.md

### PR DESCRIPTION
**Pull request by SaintAngeLs**
#254  public community issues

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,70 @@
-## Student life activity app - Mini Space
+<!DOCTYPE html>
+<html lang="en">
```
Comment (an2508374) [inline README.md:2]: This html doesn't work as we would like to.
Screenshot from 2024-06-02 12-17-20

File: README.md [hunk 2]
```diff
@@ -1,4 +1,70 @@
...
+- **User and System Oversight**: Admins have the tools to manage user issues and system functionality, ensuring smooth operation.
+- **Content Moderation**: Capabilities to monitor and manage the content to maintain a respectful and constructive community environment.
+
+## Use Cases
+- **Events**: From browsing to attending and reviewing events.
+- **Social Interactions**: Managing friends lists, sending invitations, and sharing experiences.
+- **Administration**: Handling user reports, system updates, and data analysis for continuous improvement.
+
 
 ## General description
```
Comment (an2508374) [inline README.md:69]: I think we can remove this chapter: "General description" because it will duplicate some info.

File: README.md [hunk 3]
```diff
@@ -11,20 +77,24 @@
 - [ ] Runs in the same security sandbox as JavaScript frameworks like Angular, React or Vue
```
Comment (an2508374) [inline README.md:77]: We have Blazor Server now at frontend side, we want to update it? Or we will later make a transition to Blazor WebAssembly, so there will be no point in updating this chapter?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,59 +1,46 @@
...
+- Write detailed bug reports with a clear background and steps to reproduce.
+- Pull requests should be clear and concise, mentioning the intent of changes and any related issues.
 
-* You can use `clang-format` for C and C++.
-* Make sure to follow [PEP8](https://pep8.org/) for Python.
-* For other languages, provide a linting configuration.
+## Licensing
 
-## License
+Any contributions you make will be under the same [MIT License](LICENSE) that covers the project, ensuring that MiniSpace remains open and free for the community.
```
Comment (an2508374) [inline CONTRIBUTING.md:40]: I'm not sure about it but I think we use Apache License and its files are named "LICENCE" and the link doesn't work.

File: README.md [hunk 5]
```diff
@@ -35,4 +77,33 @@
...
+Stay informed about the code quality with our integrated Codecov badge that displays the current code coverage percentage:
+[![codecov](https://codecov.io/gh/SaintAngeLs/distributed_minispace/graph/badge.svg?token=SW3T9CN2QS)](https://codecov.io/gh/SaintAngeLs/distributed_minispace)
+
+## Contributing
+Contributions are what make the open-source community such an amazing place to learn, inspire, and create. Any contributions you make are **greatly appreciated**.
+
+Please refer to [CONTRIBUTING.md](./CONTRIBUTING.md) for more details.
+
+## License
+Distributed under the MIT License. See [LICENSE](./LICENSE) for more information.
```
Comment (an2508374) [inline README.md:93]: This links to license doesn't work, probably because of LICENCE file name.

File: README.md (hunk 1 above)
Comment (SaintAngeLs) [reply README.md]: Github html interpreter is not so good, unfortunately.

File: README.md (hunk 2 above)
Comment (SaintAngeLs) [reply README.md]: Yeap, I'd agree.

File: CONTRIBUTING.md (hunk 4 above)
Comment (SaintAngeLs) [reply CONTRIBUTING.md]: Now the link to licence should work properly.

Comment (SaintAngeLs) [reply CONTRIBUTING.md]: Yeap, the Apache licence is required there.

File: README.md (hunk 5 above)
Comment (SaintAngeLs) [reply README.md:87]: Now the link to licence should work properly.

Comment (an2508374) [reply README.md:87]: Yeah, that one in Contributing.md file works now, but we have one more at the bottom of Readme.md file.

**Review by an2508374 — APPROVED** [review]

File: README.md (hunk 5 above)
Comment (SaintAngeLs) [reply README.md]: Very nice attention, thank you. Now it should link to the LICENCE file in the README.md correctly.

**Pull request merged**